src/operator/slice_channel.cc

```diff
@@ -29,33 +29,45 @@ DMLC_REGISTER_PARAMETER(SliceChannelParam);
 MXNET_REGISTER_OP_PROPERTY(SliceChannel, SliceChannelProp)
 .describe(R"code(Splits an array along a particular axis into multiple sub-arrays.
 
-Assume the input array has shape ``(d_0, ..., d_n)`` and we slice it into *m*
-(``num_outputs=m``) subarrays along axis *k*, then we will obtain a list of *m*
-arrays with each of which has shape ``(d_0, ..., d_k/m, ..., d_n)``.
+.. note:: ``SliceChannel`` is depreacted. Use ``split`` instead.
 
-For example::
+**Note** that `num_outputs` should evenly divide the length of the axis 
+along which to split the array.
 
-  x = [[1, 2],
-       [3, 4],
-       [5, 6],
-       [7, 8]]  // 4x2 array
+Example::
 
-  y = split(x, axis=0, num_outputs=4) // a list of 4 arrays
-  y[0] = [[ 1.,  2.]]  // 1x2 array
+   x = [[1, 2],
+        [3, 4],
+        [5, 6],
+        [7, 8]]
+   x.shape = (4, 2)
+   y = split(x, axis=0, num_outputs=4) // a list of 4 arrays
+   y = [[ 1.  2.]],
+       [[ 3.  4.]],
+       [[ 5.  6.]],
+       [[ 7.  8.]]
 
-  z = split(x, axis=0, num_outputs=2) // a list of 2 arrays
-  z[0] = [[ 1.,  2.],
-          [ 3.,  4.]]
+   y[0].shape = (1, 2)
 
-When setting optional argument ``squeeze_axis=1``, then the *k*-dimension will
-be removed from the shape if it becomes 1::
+   z = split(x, axis=0, num_outputs=2) // a list of 2 arrays
+   z = [[ 1.  2.]
+        [ 3.  4.]],
+       [[ 5.  6.]
+        [ 7.  8.]]
 
-  y = split(x, axis=0, num_outputs=4, squeeze_axis=1)
-  y[0] = [ 1.,  2.]  // (2,) vector
+   z[0].shape = (2, 2)
+
+`squeeze_axis=1` removes the dimension of length 1 along the `axis` 
```

Comment:
Hi @nswamy - the axis has length one. not the dimension. 

perhaps "Removes the axis with length 1 from the array shape."